### ISSUE TITLE
iOS 13: Voice Control does not work reliably

### ISSUE THREAD

**Issue by friedbunny** [mapbox/mapbox-gl-native#14832]
Testing out Voice Control with iOS 13.0b1:

- Commands like “zoom in” seem to work, but subsequent “repeat _n_ times” may be refused (for unknown reasons, but Studio Preview is refused while Treble is not).
- Voice Control itself does not perform well and can get an app into a bad state — using the repeat phrase multiple times ended up making Treble unresponsive and ultimately crashed the app.

/cc @1ec5

**Comment by 1ec5** [comment]
> Commands like “zoom in” seem to work, but subsequent “repeat n times” may be refused (for unknown reasons, but Studio Preview is refused while Treble is not).

I wonder if it has anything to do with where the focus ends up after the first zoom in. Studio Preview has a different view hierarchy than Treble. Perhaps in Studio Preview the focus could end up on a superview instead of the map view.

> Voice Control itself does not perform well and can get an app into a bad state — using the repeat phrase multiple times ended up making Treble unresponsive and ultimately crashed the app.

Possibly related to #14050 or #14324.
